src/PoolManager.sol

```diff
@@ -264,13 +261,15 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
     }
 
     /// @inheritdoc IPoolManager
-    function sync(Currency currency) public returns (uint256 balance) {
-        balance = currency.balanceOfSelf();
-        currency.setReserves(balance);
+    function sync(Currency currency) external {
+        Reserves.checkNotSynced();
+        if (currency.isNative()) return;
```

Comment:
This could make it easier for people to work with, since they can stick to the sync->settle flow, even for native tokens.
But if you think it should better revert in that scenario, let me know!

Reply:
Im happy with it returning instead. Theres no need for it to revert imo